### ISSUE TITLE
ESM error with @rdf-esm/namespace

### ISSUE THREAD

**Issue by brechtvdv** [tpluscode/rdf-ns-builders#22]
Hi,

I'm using the clownface-shacl-path NPM library, which uses this library.
I'm getting following error when compiling my code:
```
C:\Users\Administrator\Documents\types\node_modules\@tpluscode\rdf-ns-builders\vocabularies\acl.js:7
const namespace_1 = __importDefault(require("@rdf-esm/namespace"));
                                    ^

Error [ERR_REQUIRE_ESM]: require() of ES Module C:\Users\Administrator\Documents\types\node_modules\@rdf-esm\namespace\index.js from C:\Users\Administrator\Documents\types\node_mod
ules\@tpluscode\rdf-ns-builders\vocabularies\acl.js not supported.
Instead change the require of index.js in C:\Users\Administrator\Documents\types\node_modules\@tpluscode\rdf-ns-builders\vocabularies\acl.js to a dynamic import() which is available in all CommonJS modules.
```

Do you have any idea how to fix this?

Kind regards,
Brecht

**Comment by tpluscode** [comment]
Hello @brechtvdv. Sorry about that. `@rdf-esm/namespace` is now only ESM but it's strange that a newer version got installed. It should be using `v0.5.3` but apparently you got `v0.6`

Please share the output of `npm ls @rdf-esm/namespace` or `yarn why @rdf-esm/namespace`

**Comment by brechtvdv** [comment]
It is v0.5.3:
```
ngsi-ldes@0.0.1 C:\Users\Administrator\Documents\ngsi-ldes
+-- @rdf-esm/namespace@0.5.3 extraneous
+-- @tpluscode/rdf-ns-builders@1.1.0 extraneous
| `-- @rdf-esm/namespace@0.5.3 deduped
`-- @treecg/types@0.2.2 -> .\..\types
  `-- clownface-shacl-path@1.3.0
    `-- @tpluscode/rdf-ns-builders@1.1.0
      `-- @rdf-esm/namespace@0.5.4
```

**Comment by tpluscode** [comment]
Ok, I see what happens. I accidentally published a broken `v0.5.4` (notice the last line). I thought it was enough to deprecate but that's not how npm works.

Please reinstall now and make sure you get version 0.5.5 which reverts back to state of 0.5.3

**Comment by brechtvdv** [comment]
Great it works! 
This fixed it? https://github.com/rdf-esm/namespace/commit/71b923cddb2b06153ba4f874e548852c15b099aa#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346

**Comment by tpluscode** [comment]
Yes. I accidentally published 0.5.4 which should have been 0.6 (ESM-only)
